### DISCUSSION TITLE
Hop Server setup

### DISCUSSION THREAD

**Discussion by marvinhorst** [apache/hop#4684] [Q&A]
I have several Hop projects where I'm running workflows and pipelines via CLI with cron. This is all working great. I need to execute a few workflows in different projects remotely via REST calls. I've successfully gotten Hop Server running but am apparently a little thick headed because I can't seem to get any workflows or pipelines to execute on Hop Server. I'm probably missing something simple. Here's an an example REST call that just returns a 404 .  

`curl --location --request GET 'http://biapp-ltz-01:8081/hop/workflow/start' \
--header 'Content-Type: application/json' \
--header 'Authorization: ••••••' \
--data '{
  "projectName": "corporate_dw",
  "environmentName": "corporate_dw-PROD",
  "workflowName": "TimeEntries",
  "workflowFilename": "paylocity/TimeEntries.hwf",
  "runConfiguration": "default",
  "parameters": {
  }
}'`

This is my hop_server_config.xml 
<hop-server-config>
  <hop-server>
    <name>biapp-8081</name>
    <hostname>localhost</hostname>
    <port>8081</port>
  </hop-server>
  <metadata_folder>/opt/hopdata/projects/</metadata_folder>

</hop-server-config>

Do I need to configure something in my workflows and pipelines in order for them to work?

**Comment by marvinhorst** [comment]
Sorry the XML config got messed up.
```html
<hop-server-config>
  <hop-server>
    <name>biapp-8081</name>
    <hostname>localhost</hostname>
    <port>8081</port>
  </hop-server>
  <metadata_folder>/opt/hopdata/projects</metadata_folder>

</hop-server-config>
```

**Comment by hansva** [comment]
I suggest reading this discussion, it covers the steps needed to launch a pipeline via the Rest API.
https://github.com/apache/hop/discussions/4634

I am not sure where you got that endpoint from it's not listed in https://hop.apache.org/manual/latest/hop-server/rest-api.html and as far as I know does not exist
